src/annotator/guest.js

```diff
@@ -672,4 +673,31 @@ export default class Guest {
     this.visibleHighlights = shouldShowHighlights;
     this._emitter.publish('highlightsVisibleChanged', shouldShowHighlights);
   }
+
+  /**
+   * Return the scrollable element that contains the main document content.
+   *
+   * @return {HTMLElement}
+   */
+  contentContainer() {
+    if (this.pdfIntegration) {
+      return this.pdfIntegration.contentContainer();
+    } else {
+      return this.element;
```

Comment:
In future these two code paths could be combined into something like `this.integration.contentContainer()` where `this.integration` is either the HTML or PDF (or ebook etc.) integration.